k6.js

```diff
@@ -0,0 +1,44 @@
+import {
+  check, group, sleep, fail,
+} from 'k6';
+import { Rate } from 'k6/metrics';
+import http from 'k6/http';
+
+// let's collect all errors in one metric
+//const errorRate1 = new Rate('error_rate');
+//const errorRate2 = new Rate('error_rate');
+const errorRate3 = new Rate('error_rate');
+
+// See https://k6.io/docs/using-k6/options
+export const options = {
+  thresholds: {
+    error_rate: ['rate < 0.1'],
+    http_req_duration: ['p(95)<100'], // 99% of requests must complete below 1.5s
+  },
+  stages: [
+    { duration: '5s', target: 1500 }, // below normal load
+    { duration: '20s', target: 1500 },
+    // { duration: '30s', target: 1000 }, // normal load
+    // { duration: '1m', target: 500 },
+    // { duration: '30s', target: 200 }, // scale down. Recovery stage.,
+  ],
+  ext: {
+    loadimpact: {
+      distribution: {
+        Dublin: { loadZone: 'amazon:ie:dublin', percent: 100 },
```

Comment:
Getting ready for AWS ?

Reply:
@emmanuelmota that was the default, didn't even notice that